docs/user/general/datatypes.rst

```diff
@@ -155,15 +155,15 @@ The following matrix illustrates the conversions allowed by our query engine for
 +--------------+------+-------+---------+------+-------+--------+---------+--------------+------+--------+-----------+------+------+----------+----------+-----------+-----+--------+-----------+---------+
 |    STRING    |   E  |   E   |    E    |   E  |   E   |    E   |    IE   |       X      |   X  |   N/A  |     IE    |   IE |   IE |     IE   |     X    |     X     |  X  |    X   |     X     |    X    |
```

Comment:
Is it stated anywhere what `E`, `IE`, and `X` stand for? Also what is the implication of a blank space and how is it  different than having an `X`?

Reply:
Yes, please see under that table. The wording is not very clear, so if you have ideas how to improve it - welcome!

Reply:
It was just cut of from my view, after reviewing it the table makes perfect sense, thank you.